src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title: Examples of high-performance CSS animations
+authors:
+  - rachelandrew
+description: |
+  Demonstrating how high performance techniques can create complex and beautiful animations.
+date: 2020-10-20
+tags:
+  - animations
+  - performance
+---
+
+In this post find out how some popular animations found on CodePen have been created.
+THese animations all use the performant techniques discussed in other articles in this section.
+
+See [Why are some animations slow?](/animations-overview/) to learn the theory behind
+these recommendations, and the [Animations Guide](/animations-guide) for practical tips.
+
+## Wizard loading animation
+
+{% IFrame {
+  src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
+  title: 'Only CSS Loader - Wizard by Guilmain Dorian'
+}
+%}
+
+This loading animation is built entirely with CSS.
+The image plus all of the animation has been created in CSS and HTML,
+no images or JavaScript.
+To understand how it was created and how well it performs you can use Chrome DevTools.
+
+With the animation running, open the Performance tab in Chrome DevTools and record a few seconds of the animation.
```

Comment:
We should link to the full CodePen because we're suggesting to readers to actually try out this workflow: https://codepen.io/iamlark/full/jYzYJg

FYI one of my main issues with CodePen (and one of the reasons Glitch was so appealing) was that CodePen would run the samples in iframes (whereas Glitch just gives you your running code with no other cruft). In this case it doesn't seem to big of an issue.

Reply:
On the other hand, now that the Performance panel segments main thread activity by frames, perhaps it's not too big of a deal anymore:

![image](https://user-images.githubusercontent.com/4713486/96823712-ed048d80-13e1-11eb-9152-8071318d64e5.png)